mastercraft.cmd

```diff
@@ -670,7 +685,7 @@ calc.material:
                var herb2 NULL
                var herb1.volume 5
                if %order.chapter = 2 then var herb2.volume 1
-               if matchre("%full.order.noun", "some blister cream|some moisturizing ointment|some itch salve|some lip balm") then var herb1 red.flowers
+               if matchre("%full.order.noun", "some blister cream|some moisturizing ointment|some itch salve|some lip balm") then var herb1 flowers

```

Comment:
These lines look EXACTLY the same to my eyes. I am not sure why they are getting picked up as changed.